distribution/sender/routes/sender-activemq-consumer.xml

```diff
@@ -0,0 +1,93 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="sender-activemq-consumer" errorHandlerRef="shutdownErrorHandler">
+        <from uri="activemq:openmrs.sync.{{db-sync.senderId}}?connectionFactory=activeMqConnFactory&amp;acknowledgementModeName=CLIENT_ACKNOWLEDGE&amp;messageListenerContainerFactory=customMessageListenerContainerFactory&amp;asyncStartListener=true" />
+
+        <log message="Received message -> ${body}" />
+        
+        <setProperty name="requestUuid">
+            <jsonpath suppressExceptions="true">$.requestUuid</jsonpath>
```

Comment:
What if the requestUuid exists but the unmarshalling fails because of some other issue? Don't you think it would you are masking a problem and might process the message wrongly?